source/module_base/mathzone_add1.cpp

```diff
@@ -279,6 +280,7 @@ void Mathzone_Add1::Uni_Deriv_Phi
     fftw_complex *fft_phik = new fftw_complex[FFT_NR];
     fftw_complex *fft_ndphik = new fftw_complex[FFT_NR];
     fftw_complex *fft_ndphir = new fftw_complex[FFT_NR];
+	ModuleBase::Memory::record("Mathzone_Add1::Uni_Deriv_Phi",sizeof(fftw_complex) * FFT_NR * 4);
```

Comment:
this line may not be necessary since the array will be deleted when the function ends